src/test/java/com/coherent/task40/Customer.java

```diff
@@ -0,0 +1,70 @@
+package com.coherent.task40;
+
+import org.openqa.selenium.By;
+
+public class Customer {
+    public String name;
+    public String position;
+    public String office;
+    public String age;
+    public String salary;
```

Comment:
Access modifier should be private

Reply:
updated

Reply:
I'm wondering why can't we leave them public?